.github/workflows/publish-docker.yml

```diff
@@ -43,7 +47,7 @@ jobs:
           build_timestamp=$(date '+%s')
           docker buildx build --platform=linux/amd64,linux/arm64 \
             -f ${{ github.event.inputs.dockerfile || 'Dockerfile' }} \
-            -t "nethermindeth/nethermind:${{ github.event.inputs.tag || 'master' }}" \
+            -t "${{ github.event.inputs.image-name }}:${{ github.event.inputs.tag || 'master' }}" \
```

Comment:
Please ensure the Docker Hub repo name is always hardcoded so no one enters `nethermind` instead of `nethermindeth` by mistake. Thus:

```
nethermindeth/${{ github.event.inputs.image-name }}
```